views/addmarker.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
```

Comment:
I don't see this file being used at all, and I thought that we planned to use a modal to handle adding markers instead of a separate page.